changelog.d/splunk_hec_sink_encode_message_only.breaking.md

```diff
@@ -0,0 +1,2 @@
+The Splunk HEC sink is now using the `message` meaning to retrieve the relevant value and encodes the retrieved value
+only. Note that if the retrieved value is `None`, an event with an empty message will be published.
```

Comment:
A few questions:

- Do we know how Splunk reacts if you send it an event with an empty `event` field?
- Is the semantic field required? I.e. will Vector fail to boot if there is no semantic `message` defined?
- Also, is this only true for the `/event` endpoint and not for `/raw`?